### PR TITLE
Add tests file for the bisect module.

### PR DESCRIPTION
**Pull request by DimitrisJim**
Kept the same structure (as also described in [PEP 399](https://www.python.org/dev/peps/pep-0399/)) in order to easily be able to plug in a future rust implementation of bisect.

No changes other than removing statements dealing with the C accelerator module (and importing module directly for now). All tests locally pass.

### CODE REVIEW

**Comment by fanninpm** [comment]
@coolreader18 should this commit be split (as per [your comment on a separate issue from February 2021](https://github.com/RustPython/RustPython/pull/2455#issuecomment-778901045))?

**Comment by coolreader18** [comment]
Oh, yeah, that would be good. @DimitrisJim could you have one commit that's *just* copying the file directly from CPython (Like "Add test.test_bisect from CPython x.y" for whatever CPython version you used), and another with the changes you make?

**Comment by DimitrisJim** [comment]
@coolreader18 done. Wasn't aware of the etiquette on this (licensing related?). Hopefully everything is fine now.

**Review by coolreader18 — APPROVED** [review]
Yup, looks good! I'm not exactly sure about licensing, but it also helps with seeing the history of the changes we've made to test files; what needed to be skipped and so forth.

File: Lib/test/test_bisect.py [hunk 1]
```diff
@@ -290,10 +282,7 @@ def test_arg_parsing(self):
             self.assertRaises(TypeError, f, 10)
 
 class TestErrorHandlingPython(TestErrorHandling, unittest.TestCase):
-    module = py_bisect
-
-class TestErrorHandlingC(TestErrorHandling, unittest.TestCase):
-    module = c_bisect
+    module = bisect
```
Comment (coolreader18) [inline Lib/test/test_bisect.py:285]: Another option for this is to leave the `py_bisect = `/`c_bisect =`/`module = py_bisect` as is, and just add like `@unittest.skip("TODO: RUSTPYTHON, _bisect module")` to the C tests.

Comment (DimitrisJim) [reply Lib/test/test_bisect.py:285]: Ah, yes. The reason for that is `test.support.import_fresh_module` isn't currently implemented. I could just change it to:

```python
...
import bisect
...

py_bisect = bisect
c_bisect = None
```

and then annotate the classes with `skip`. Adding skip is definitely a better option as it acts like a marker.

**Comment by DimitrisJim** [comment]
@coolreader18 done! Thanks for the feedback.

**Comment by DimitrisJim** [comment]
I noticed I was being daft, `support.import_fresh_module` exists just fine. (I guess it errored previously because `_bisect` doesn't exist and I must've misread the error.)

@coolreader18 do you think we can push this? (I'm waiting on it to push a `_bisect` implementation I'm working on).

**Comment by coolreader18** [comment]
Oh, yeah, sorry about that!

**Pull request merged**